pkg/naming/names.go

```diff
@@ -100,6 +100,16 @@ func IndexFromName(n string) (int32, error) {
 	return int32(index), nil
 }
 
+func RackOrdinalFromName(name string, sc *scyllav1.ScyllaCluster) (int, error) {
```

Comment:
sounds like `FindIndex` that could share implementation with https://github.com/scylladb/scylla-operator/blob/18e1ecc18233a952560ccac46507115b93911a77/pkg/helpers/array.go#L72-L80

Reply:
done